html/index.html

```diff
@@ -116,22 +116,22 @@
                             <h3 class="title is-size-4-desktop is-size-5">Enter address and amount (0.001-0.1)</h3>
 
                             <div id="faucet_err"></div>
-
+                            <form id="myform" action="javascript:faucet_fetch();" autocomplete="off">
                             <div class="field has-margin-top-25">
                                 <div class="control">
-                                    <input class="input" id="address" name="address" type="text" placeholder="tb1q...">
+                                    <input class="input" id="address" name="address" type="text" placeholder="tb1q..." tabindex="1" autocapitalize="off" autocorrect="off" pattern="^tb1q.{8,}" title="Use a valid signet address.">
```

Comment:
Note that some addresses start with `tb1p`.

Reply:
OK, I just re-used what was there in the code. Changing...

Reply:
Ah you mean the placeholder? Yeah, that was done pre-taproot. Should probably be `tb1...` at this point.

Reply:
See 1e25bd3 please.